optaweb-vehicle-routing-backend/src/test/java/org/optaweb/vehiclerouting/plugin/routing/AirDistanceRouterTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaweb.vehiclerouting.plugin.routing;
+
+import org.junit.jupiter.api.Test;
+import org.optaweb.vehiclerouting.domain.Coordinates;
+import org.optaweb.vehiclerouting.service.region.BoundingBox;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class AirDistanceRouterTest {
+
+    @Test
+    void travel_time_should_be_distance_divided_by_speed() {
+        AirDistanceRouter router = new AirDistanceRouter();
+        Coordinates from = Coordinates.valueOf(0, 0);
+        Coordinates to = Coordinates.valueOf(3, 4); // √(3² + 4²) = 5
+        long travelTimeMillis = router.travelTimeMillis(from, to);
+        assertThat(travelTimeMillis).isEqualTo((long) (5
+                * AirDistanceRouter.KILOMETERS_PER_DEGREE
+                / AirDistanceRouter.TRAVEL_SPEED_KPH
+                * 60 * 60 * 1000));
```

Comment:
where did this constant come from?

Reply:
It's number of milliseconds in one hour. It's needed to convert travel time in hours (kilometers / speed in kph) to travel time in milliseconds. I've just noticed the literal value is duplicated between the main and test sources so +1 to isolating to a named constant. Thanks!